cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,3 +1,7 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+{% set e2e_opts = '' -%}
+{% if pillar.get('e2e_test_environment', '').lower() == 'true' -%}
+{% set e2e_opts = '-s devicemapper' -%}
```

Comment:
that said, it seems like coupling e2e_opts to the storage driver is a little too subtle.

can we just have a separate flag in salt that controls devicemapper only?  then have a config-e2e.sh that uses that option if required?